js/script.js

```diff
@@ -0,0 +1,35 @@
+const taskInput = document.getElementById("task-input");
+const addButton = document.getElementById("add-button");
+const unorderedTaskList = document.getElementById("tasks");
+const toaster = document.getElementById("message");
```

Comment:
1. It's generally a good convention to prepend `$` to variable names which contains DOM elements

2. All the tasks in the list will be ordered in terms of their creation dates. So we can fallback to a generic `taskList` name.

```suggestion
const $taskInput = document.getElementById("task-input");
const $addButton = document.getElementById("add-button");
const $taskList = document.getElementById("tasks");
const $toaster = document.getElementById("message");
```

Reply:
refactored these issues.